sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -148,6 +148,12 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ONLINE_TAKE_ENABLED = SQLConfigBuilder("spark.sql.onlineTake.enabled")
```

Comment:
This is the first name that came to mind for this feature-flag, but I'm sure there's a clearer alternative. Please suggest.